### ISSUE TITLE
Interactive plot doesn't work when y axis is on log scale

### ISSUE THREAD

**Issue by astefan1** [nicebread/corona#17]
If you select the option to display the y-axis, the lines in the interactive plot disappear. Just writing this down so that we don't forget, can try to fix this myself.

**Comment by zuphilip** [comment]
That is right and I have seen that before. I haven't looked into this. Any help to fix this is very welcome!

**Comment by nicebread** [comment]
For the moment, the app hides and disables the logScale when plotly is activate (see f3f87b512f920e5507eb932adbe48357adce142d)
